packages/i18n-calypso/package.json

```diff
@@ -4,6 +4,7 @@
 	"description": "i18n JavaScript library on top of Jed originally used in Calypso",
 	"main": "dist/cjs/index.js",
 	"module": "dist/esm/index.js",
+	"types": "types/index.d.ts",
```

Comment:
This change and `packages/i18n-calypso/types/index.d.ts` was meant for another PR perhaps?

Reply:
Never mind, saw f38f66bde6 only now. :-)

Reply:
It's from #33646, mentioned in the commit message but I didn't call it out, thanks.

I'd remove them before landing this.